docs/HANDLED_RULES.md

```diff
@@ -11,6 +11,7 @@ Sonarqube-repair can currently repair violations of 12 rules of which 10 are lab
     * [JEE applications should not "getClassLoader"](#jee-applications-should-not-getclassloader-sonar-rule-3032) ([Sonar Rule 3032](https://rules.sonarsource.com/java/RSPEC-3032))
     * ["compareTo" should not return "Integer.MIN_VALUE"](#compareto-should-not-return-integermin_value-sonar-rule-2167) ([Sonar Rule 2167](https://rules.sonarsource.com/java/RSPEC-2167))
     * [Math should not be performed on floats](#math-should-not-be-performed-on-floats-sonar-rule-2164) ([Sonar Rule 2164](https://rules.sonarsource.com/java/RSPEC-2164))
+    * [Synchronization should not be based on Strings or boxed primitives](#Synchronization-should-not-be-based-on-Strings-or-boxed-primitives) ([Sonar Rule 1860](https://rules.sonarsource.com/java/tag/multi-threading/RSPEC-1860))
```

Comment:
Correct link: synchronization-should-not-be-based-on-strings-or-boxed-primitives-sonar-rule-1860

Reply:
Henry, the link is still wrong.